pages/common/date.md

```diff
@@ -21,3 +21,9 @@
 - Convert a specific date to the Unix timestamp format:
 
 `date -d "{{2018-09-01 00:00}}" +%s --utc`
+
+- Display RFC-3339, great for logging and timestamps YYYY-MM-DD hh:mm:ss TZ
```

Comment:
```suggestion
- Display RFC-3339 logging with timestamps:
```
Much simpler this way

Reply:
Hrm, @zdroid this does not convey the same meaning. "Display RFC-3339 logging" does not make much sense, the `date` command has nothing to do with logging *per se*, and neither does the RFC-3339, it's just a date format. Other than that, keeping the date format makes sense IMHO, that way you don't have to try the command to know the format. I would suggest the following:

```suggestion
- Display the current date using the RFC-3339 format (`YYYY-MM-DD hh:mm:ss TZ`):
```

Which is also the same way other examples in the page are worded.

Reply:
Yes, you're right. I tried to rephrase the original, but I realized it has nothing to do with logging after running the command.